### PR TITLE
Fixing OSX build error

### PR DESCRIPTION
**Pull request by issacnitin**

### CODE REVIEW

File: opencog/nlp/irc/IRC.cc [hunk 1]
```diff
@@ -149,7 +149,7 @@ int IRC::start(const char* server, int port, const char* nick,
 
 	/* Ping every .. I dunno -- 5 minutes? */
 	optval = 300;
-	rc=setsockopt(irc_socket, IPPROTO_TCP, TCP_KEEPIDLE, &optval, optlen);
+	rc=setsockopt(irc_socket, IPPROTO_TCP, TCP_KEEPCNT, &optval, optlen);
 	if (0 > rc)
```
Comment (linas) [inline opencog/nlp/irc/IRC.cc]: Hmm. Actually, both the original, and your change are wrong; it should have been `TCP_KEEPINTVL` -- does MacOS support that?

**Comment by linas** [comment]
Anyway, I just now changed it to `TCP_KEEPINTVL`

File: opencog/nlp/irc/IRC.cc (hunk 1 above)
Comment (issacnitin) [reply opencog/nlp/irc/IRC.cc]: Yes it does, thanks.

**Comment by ngeiswei** [comment]
Can we close this PR?

**Comment by ngeiswei** [comment]
I assume we can.

**Pull request closed without merging**